### PR TITLE
rebuild ODY data - fixes #382

### LINKED ISSUE #382: 35 cards are missing in Odyssey
Hello,

The following cards are missing in Odyssey:
```
Magnivore
Malevolent Awakening
Master Apothecary
Metamorphic Wurm
Millikin
Mind Burst
Mindslicer
Mine Layer
Minotaur Explorer
Mirari
Molten Influence
Moment's Peace
Morbid Hunger
Morgue Theft
Mortivore
Mossfire Egg
Mossfire Valley
Mudhole
Muscle Burst
Mystic Crusader
Mystic Enforcer
Mystic Penitent
Mystic Visionary
Mystic Zealot
Nantuko Disciple
Nantuko Elder
Nantuko Mentor
Nantuko Shrine
Need for Speed
Nefarious Lich
New Frontiers
Nimble Mongoose
+ Mountain 1,2,3,4
```

### PR DESCRIPTION
**Pull request by gwax**
rebuild ODY data: fixes #382

### CODE REVIEW

**Comment by fenhl** [comment]
Update the changelog please.

**Comment by gwax** [comment]
@fenhl already on it (and done)

File: web/changelog.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 [
   {
+    "version": "3.8.x",
```
Comment (fenhl) [inline web/changelog.json]: This seems unsuitable for release. Maybe we should start working with release branches?

**Comment by fenhl** [comment]
Travis error seems unrelated.

File: web/changelog.json (hunk 1 above)
Comment (gwax) [inline web/changelog.json]: I figure it's nice to have a placeholder for people to add things to without necessarily needing to know what the ultimate release number will be.

Comment (fenhl) [inline web/changelog.json]: True, but with our current setup, a pull request based on master will be released as is on merge.

**Pull request merged**